cmd/main.go

```diff
@@ -145,7 +168,43 @@ func parseCliArgs(flags *args) (*cobra.Command, error) {
 	return rootCmd, nil
 }
 
-func run(kubeconfigArgs *genericclioptions.ConfigFlags, flags args, printFlags *k8sget.PrintFlags) (int, error) {
+func loadConfig(configPath string) (conf Config, err error) {
+	viper.SetConfigFile(configPath)
+	viper.SetConfigType("yaml")
+
+	viper.AutomaticEnv()
+
+	err = viper.ReadInConfig()
+	if err != nil {
+		return
+	}
+
+	err = viper.Unmarshal(&conf)
+	return
+}
+
+func validateConfig(conf Config) error {
+	for i, exclusion := range conf.Exclusions {
+		if exclusion.Description == nil {
+			return fmt.Errorf("description must not be empty for exlusion rule %d", i)
+		}
+		if exclusion.Name == nil {
+			return fmt.Errorf("name must not be empty for exlusion rule %d", i)
+		}
+		if exclusion.Kind == nil {
+			return fmt.Errorf("kind must not be empty for exlusion rule %d", i)
+		}
```

Comment:
kind.kind feels a bit odd,  I think grouping the the api stuff into kind is not needed, it should be at the same level:

```yaml
exclusions:
- name: foo-
  kind: MyResource
```